packages/test/test-utils/src/testContainerRuntimeFactoryWithDefaultDataStore.ts

```diff
@@ -0,0 +1,81 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { ContainerRuntimeFactoryWithDefaultDataStore } from "@fluidframework/aqueduct";
+import { IContainerRuntimeOptions } from "@fluidframework/container-runtime";
+import { IContainerRuntime } from "@fluidframework/container-runtime-definitions";
+import { RuntimeRequestHandler } from "@fluidframework/request-handler";
+import {
+	IFluidDataStoreFactory,
+	NamedFluidDataStoreRegistryEntries,
+} from "@fluidframework/runtime-definitions";
+import { UsageError } from "@fluidframework/telemetry-utils";
+
+const getDefaultFluidObject = async (runtime: IContainerRuntime) => {
+	const entryPoint = await runtime.getAliasedDataStoreEntryPoint("default");
+	if (entryPoint === undefined) {
+		throw new UsageError("default dataStore must exist");
+	}
+	return entryPoint.get();
+};
+
+export const createTestContainerRuntimeFactoryWithDefaultDataStore = (
```

Comment:
This is a sort of weird thing needed if we want to make `initializeEntryPoint` required on the `ContainerRuntimeFactoryWithDefaultDataStore` class.
Alternatively, we could leave it as optional on that class and provide this `getDefaultFluidObject` method as the default.

Reply:
I think `ContainerRuntimeFactoryWithDefaultDataStore` should not have `initializeEntryPoint` as an argument. It accepts a defaultFactory param and creates an aliased data store off of that during creation. That is what should be returned as the entry point.

Reply:
I think `initializeEntryPoint` will need to remain on this ctor as long as we have `requestResolvedObjectFromContainer`. Tests on that method expect the entryPoint to act as container runtime request router (see `loaderTest.spec.ts`)